akka-actor-tests/src/test/scala/akka/util/VersionSpec.scala

```diff
@@ -35,5 +33,12 @@ class ManifestInfoVersionSpec extends AnyWordSpec with Matchers {
       new Version("1.2-foo") should be > new Version("1.2.3")
```

Comment:
Hmmm, so `1.2-alpha` is newer than `1.2.3`?

Reply:
Of course users that deployed `1.2-alpha` can just deploy `1.3.0` and move on. My concern is that they discover that version `1.2.3` is considered older than `1.2-alpha` when deploying already.

Reply:
`1.2-alpha` should probably be 1.2.0-alpha. I think maven would have same problem otherwise.

I think the reason for this could be `1.2-SNAPSHOT` > `1.2.0`

Reply:
WDYM? `1.2-SNAPSHOT` identifies artifacts produced _before_ `1.2.0` (https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm#MAVEN401).

If `1.2.0` is out, then `1.2-SNAPSHOT` must not be used. It is ambiguous. There may be `1.2.1-SNAPSHOT`  and `1.3-SNAPSHOT`.

Reply:
fixed